static/css/page-form.less

```
@@ -144,3 +144,76 @@ div#revertLink,
     float: left;
   }
 }
+
+// Styling for the new cover form accepting Copy/Pasted images
+
+.ol-cover-form {
+  margin-bottom: 20px;
+
+  .label {
+    margin-bottom: 10px;
+  }
+
+  .input {
+    display: flex;
+    gap: 10px;
+    align-items: center;
+  }
+
+  &--ia {
+    &_image {
+      height: 200px;
+    }
+  }
+
+  &--clipboard {
+    #image {
+      max-width: 100%;
+      max-height: 400px;
+      display: none;
+
+      &.visible {
+        display: block;
+      }
```

```suggestion
    .image-container img {
      max-width: 100%;
      max-height: 400px;
```
Creating the `img` client-side removes the need to toggle the image's visibility.  We're no longer setting an `id` on the image, so the selector can be updated as well.